src/Video/Texture/Texture2D.cpp

```diff
@@ -31,14 +31,6 @@ GLuint Texture2D::GetTextureID() const {
     return texID;
 }
 
-int Texture2D::GetWidth() const {
-    return width;
-}
-        
-int Texture2D::GetHeight() const {
-    return height;
-}
-
 void Texture2D::Load(const char* filename, bool srgb) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetWidth' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)